digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/TrafficSignCsvImporter.scala

```diff
@@ -260,8 +285,27 @@ class TrafficSignCsvImporter(roadLinkServiceImpl: RoadLinkService, eventBusImpl:
             }
           case (_, _) => (false, List("Invalid dates"))
         }
-      case _ => (true, Nil)
-    }
+      }
+    //If it is another sign but have dates
+      else if ( optStartDate.head.nonEmpty || optEndDate.head.nonEmpty) {
+        (optStartDate, optEndDate) match {
+          case (Some(startDate), Some(endDate)) => //If dates have value, lets check if they are ok
+            try {
+              val startDateFormat = DateParser.DatePropertyFormat.parseDateTime(startDate)
+              val endDateFormat = DateParser.DatePropertyFormat.parseDateTime(endDate)
+
+              val isDatesOk = endDateFormat.isAfter(startDateFormat) || endDateFormat.isEqual(startDateFormat)
+              val errorMsg = if (isDatesOk) Nil else List("The end date value is equal/previous to the start date value.")
+
+              (isDatesOk, errorMsg)
+            } catch {
+              case _: Throwable => (false, List("Invalid dates formats"))
+            }
```

Comment:
This try catch can probably be moved to a function since there is code above that is exactly the same

Reply:
Fixed